mahl-web-scraper-ui/src/theme.ts

```diff
@@ -0,0 +1,6 @@
+export default {
+  colors: {
+    primary: '#720207',
```

Comment:
generally I like comments here that describe the color esp since I'm noob at hex